### DISCUSSION TITLE
a

### DISCUSSION THREAD

**Discussion by hell001** [groue/GRDB.swift#1055] [Q&A]
a

**Comment by groue** [comment]
Hello @hell001,

Support does not include writing your application for you, or teaching database design. You will have to help yourself a little more, and look for information on dedicated websites.

If you wonder how to deal with many-to-many relationships with GRDB, look for "passport" in the Association documentation. It implements a many-to-many relationship between citizens and countries. You'll see that you need a third table in the middle.

And if you have more precise and focused questions about GRDB, please ask! Support is great especially when it is possible to provide answers to single questions, that can be searched with Google or the GitHub search form, _for the benefit of all users_.

**Reply by groue** [reply]
> I don't necessarily have a completely separate third entity C that is related to A and B. The third table would just a be a bridge table containing id's from A and B Please advise.

Yes, define a record type for this third table C, and use it in the definition of the relationship between A and B.

**Reply by groue** [reply]
As for the rest of your message, I think StackOverflow will be a better place for your questions.

**Comment by groue** [comment]
Related issue: https://github.com/groue/GRDB.swift/issues/1063
